server/bin/pbench-server-activate-create-crontab

```diff
@@ -143,7 +143,8 @@ for role in $roles ;do
             done
             ;;
         pbench-prep)
-            shims=$(getconf.py -l shims $role)
+            version=$(getconf.py -l pbench-move-results-receive-versions pbench-server)
```

Comment:
ATM, there is only one version, but it is conceivable that we might end up with two in the future (the same way that we had two in the past: there is some unforeseen problem that requires us to introduce a second one). This is not going to work in that case

Reply:
Looks like this has been resolved.